analyzers/tests/SonarAnalyzer.UnitTest/SymbolicExecution/Roslyn/RoslynSymbolicExecutionTest.Loops.cs

```diff
@@ -82,6 +82,24 @@ public void Loops_InstructionVisitedMaxTwice_For_FixedCount_Expanded()
             x => x[validator.Symbol("i")].Should().HaveOnlyConstraints(ObjectConstraint.NotNull, NumberConstraint.From(10, null)));   // We can assert because LVA did not kick in yet
     }
 
+    [TestMethod]
+    public void Loops_InstructionVisitedMaxTwice_For_FixedCount_Expanded1()
```

Comment:
The "1" doesn't explain much, especially when it's a second. Something like `Negated`?

Reply:
Anyway, this looks like a duplicate of the original => make it parametrized